packages/jaeger-ui/src/components/DeepDependencies/index.css

```diff
@@ -14,11 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.Ddg--graphWrapper {
-  background: #f0f0f0;
-  bottom: 0;
+.Ddg {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
   left: 0;
-  position: fixed;
+  position: absolute;
   right: 0;
-  top: 0;
+}
+
+.Ddg--graphWrapper {
+  flex: 1;
```

Comment:
this + `flex-direction: column` is certainly clearer than before with `TOP_NAV_HEIGHT`
but what would cause there to be overflow?

Reply:
probably want to avoid the `TOP_NAV_HEIGHT` elsewhere as well, if possible.